test/steps/printing/3_then.js

```diff
@@ -134,90 +134,68 @@ export function theReturnedTableShouldHaveNoRows() {
 	return expect(returnValue).to.have.length(0);
 }
 
-export function theReturnedTableShouldHaveAHeadWithTheObjectKeys() {
+export function theReturnedTableShouldHaveARowWithTheObjectKeyValues() {
 	const { returnValue, testObject } = this.test.ctx;
-	const keys = Object.keys(testObject);
-	return expect(returnValue.options)
-		.to.have.property('head')
-		.eql(keys);
+	return Object.entries(testObject).forEach(([key, value], arrayKey) => {
+		expect({ [key]: value }).to.eql(returnValue[arrayKey]);
+	});
 }
 
-export function theReturnedTableShouldHaveARowWithTheObjectValues() {
+export function theReturnedTableShouldHaveARowWithTheObjectKeyAndStringifiedNestedValues() {
 	const { returnValue, testObject } = this.test.ctx;
-	const values = Object.values(testObject);
-	return expect(returnValue[0]).to.eql(values);
-}
-
-export function theReturnedTableShouldHaveAHeadWithTheObjectNestedKeys() {
-	const { returnValue } = this.test.ctx;
-	const keys = [
-		'root',
-		'nested.object',
-		'nested.testing',
-		'nested.nullValue',
-		'nested.keys.more',
-	];
-	return expect(returnValue.options)
-		.to.have.property('head')
-		.eql(keys);
-}
-
-export function theReturnedTableShouldHaveAHeadWithTheObjectNestedValues() {
-	const { returnValue } = this.test.ctx;
-	const values = ['value', 'values', 123, null, 'publicKey1\npublicKey2'];
-	return expect(returnValue[0]).to.eql(values);
-}
-
-export function theReturnedTableShouldHaveAHeadWithTheObjectsKeys() {
-	const { returnValue, testArray } = this.test.ctx;
-	const keys = Object.keys(testArray[0]);
-	return expect(returnValue.options)
-		.to.have.property('head')
-		.eql(keys);
-}
-
-export function theReturnedTableShouldHaveAHeadWithTheObjectsNestedKeys() {
-	const { returnValue, testArrayKeysResult } = this.test.ctx;
-	return expect(returnValue.options)
-		.to.have.property('head')
-		.eql(testArrayKeysResult);
+	return Object.entries(testObject).forEach(([key, value], arrayKey) => {
+		const strValue =
+			typeof value === 'object'
+				? Object.entries(value)
+						.map(([vKey, vValue]) => `${vKey}: ${JSON.stringify(vValue)}`)
+						.join('\n')
+				: value;
```

Comment:
Is this always a string?

Reply:
At this point, it's either string or number, which are both acceptable.